client/src/components/room/footer.tsx

```diff
@@ -21,8 +21,10 @@ const Footer: React.FC = () => {
         className="uiLayer horizontal-stack:s-2 padded:s-1 "
         style={{ position: "absolute", bottom: 0, right: 0 }}
       >
+        <NumOnline />
```

Comment:
Note — I noticed `in room` still showed up on the homepage. We should either show...

- The total number of people on the site at the moment (in all rooms)
- The number of people on the homepage
- Nothing (hide this component)

Reply:
I updated this to show the total number of people on the site in all rooms.
It's happening primarily on the server, which is now writing to a new collection: stats/num_online.
Perhaps that will be useful for other things later too. The client then reads from this.